script_runner/tasks.py

```diff
@@ -61,6 +61,12 @@
 
 IS_WINDOWS = os.name == 'nt'
 
+def safe_remove_script(script_path):
+    try:
+        os.remove(script_path)
+    except Exception:
```

Comment:
Please only except `OSError` here (ideally we'd check `e.errno==2` as well but maybe that's overkill).
(I know you're only wrapping a single statement so it's not a major issue but catching & just throwing away any `Exception` is always the wrong thing to do)